ci/incremental/build.cmd

```diff
@@ -1,5 +1,5 @@
 @rem https://github.com/numba/numba/blob/master/buildscripts/incremental/build.cmd
-call activate %CONDA_ENV%
```

Comment:
I think I mentioned before. This can't be simply removed, the environment needs to be activated in `ci/azure/windows.yml` before this script is called